src/qtism/data/storage/xml/marshalling/PrintedVariableMarshaller.php

```diff
@@ -105,15 +115,25 @@ protected function unmarshall(DOMElement $element)
                 $component->setIndex((Format::isInteger($index) === true) ? (int)$index : $base);
             }
 
-            if (Version::compare($version, '2.1.0', '>=') === true && ($delimiter = $this->getDOMElementAttributeAs($element, 'delimiter')) !== null) {
+            if (Version::compare($version, '2.1.0', '>=') === true && ($delimiter = $this->getDOMElementAttributeAs(
+                    $element,
+                    'delimiter'
+                )) !== null) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place?